admin/c2cgeoportal_admin/routes.py

```diff
@@ -3,7 +3,7 @@
 
 
 def includeme(config):
-    config.add_static_view('node_modules', 'c2cgeoportal_admin:node_modules/')
+    config.add_static_view('admin/node_modules', 'c2cgeoportal_admin:node_modules')
```

Comment:
It should have been better to use a such mechanism
https://docs.pylonsproject.org/projects/pyramid/en/latest/narr/urldispatch.html?highlight=routes#using-a-route-prefix-to-compose-applications

Reply:
Thanks, I start to try it, and it it's working it will be simpler :-)